triciaswift/tastevine-client
# Added registration view and functionality

Added registration page with some initial css

## To Test
1. Pull the ts-register branch and switch to it
2. Make sure you run `npm start` in your terminal for our client side directory
3. In the url type: `http://localhost:5173/register`
4. First use the the email tricia@swift.com
5. A popup should appear saying that email already exists
6. Next type in whatever you want but make sure the passwords don't match
7. A message should appear below the Verify password saying that the password doesn't match
8. Finally make sure all fields are fille in and your passwords math
9. Click "Get Started", you should be logged into the app and directed to the home page.
10. Check local storage to make sure the token was stored.


Closes ticket #6
